classes/order/OrderInvoice.php

```diff
@@ -129,6 +129,8 @@ public function add($autodate = true, $null_values = false)
 
     public function getProductsDetail()
     {
+        $id_lang = Context::getContext()->language->id;
```

Comment:
yes, this PR should definitely not be taken as is. I created it to try to figure out the amount of missing merges / PR merges in 1.7 coming from 1.6. It seems a few useful PR / JIRA fixes are definitely missing into the 1.7 branch, and we will need to thoroughly check what still needs to be merged from the 1.6 branch.